File: PycharmProjects/Python_Class/HW_1/HW_1_4.py
import random

print("Cuantos empleados son ?: ")
num = int(input())
print(f"Tomamos en cuenta que el salario base es igual a $30000.00 pesos")
print("***************************************************************************************")

emp_list = []

for i in range(num):
    vent1 = random.uniform(50000, 150001)
    vent2 = random.uniform(50000, 150001)
    vent3 = random.uniform(50000, 150001)

    emp_list.append((i + 1, vent1, vent2, vent3))

for emp_data in emp_list:
    id, venta1, venta2, venta3 = emp_data
    comis_ven = (venta1 + venta2 + venta3) * 0.10
    com_base = 30000
    comis_tot = comis_ven + com_base

    print(f"Empleado ID: 000{id}. Ventas: (${venta1:.3f}, ${venta2:.3f}, ${venta3:.3f}). Comision de Ventas: ${comis_ven:.3f}. Comision TOTAL con base = ${comis_tot:.3f}")
    print("")

File: IdeaProjects/Java_Class/src/HW_1/HW_1_4/HW_1_4.java
package HW_1.HW_1_4;

import java.util.Random;
import java.util.Scanner;

public class HW_1_4 {

    public static void main(String[] args){


        /*
        Una compañía de seguros tiene contratados a n vendedores. Cada uno hace tres ventas a la semana.
        Su política de pagos es que un vendedor recibe un sueldo base y un 10% extra por comisiones de sus ventas.
        El gerente de su compañía desea saber cuánto obtendrá en la semana cada vendedor por concepto de comisiones
        por las tres ventas realizadas, y cuánto tomando en cuenta su sueldo base y sus comisiones.

         */

        Scanner s = new Scanner(System.in);

        System.out.println("Cuantos empleados son ?: ");
        int num = s.nextInt();
        System.out.println("Tomamos en cuenta que el salario base es igual a $" +30000+".00 pesos");
        System.out.println("***************************************************************************************");

        Empleado[] emp_list = new Empleado[num];

        Random rand = new Random();

        double vent1 = 0;
        double vent2 = 0;
        double vent3 = 0;


        for(int i = 0; i < num; i++){
            vent1 = rand.nextDouble(50000, 150001);
            vent2 = rand.nextDouble(50000, 150001);
            vent3 = rand.nextDouble(50000, 150001);

            Empleado n = new Empleado(i+1, vent1, vent2, vent3);
            emp_list[i] = n;

        }

        for(int x = 0; x < num; x++){
            System.out.println(emp_list[x]);
            System.out.println("");

        }




    }
}
